src/app/core/components/breadcrumb/breadcrumb.component.ts

```diff
@@ -22,4 +22,7 @@ export class BreadcrumbComponent {
     }
   }
 
+  trackByFn(index: number): number {
+    return index;
+  }
```

Comment:
I don't get what it is used for.

Reply:
trackBy need for avoid rerenders which angular does by default